Open Judge System/Data/OJS.Data/IOjsDbContext.cs

```diff
@@ -58,12 +59,18 @@ public interface IOjsDbContext : IDisposable
 
         int SaveChanges();
 
+        Task<int> SaveChangesAsync();
+
         void ClearDatabase();
 
+        void DbExecuteSqlCommand(string query);
+
+        void ExecuteSqlCommandWithTimeout(string query, int timeoutInSeconds);
```

Comment:
Revert these changes. Since the temp method is unsupported, those will not be used.